### DISCUSSION TITLE
[Kavita] Kavita Blue android app?

### DISCUSSION THREAD

**Discussion by Allastratou** [Kareadita/Kavita#3145] [Ideas]
### Idea Description

Came across an android app called [Kavita Blue](https://play.google.com/store/apps/details?id=com.moongoblin.KavitaBlue&hl=en) during a google search. Also [this github link without too much info](https://github.com/MoonGoblinDev/KavitaBlue). Was wondering if this was a legit app, and if so, if it could be tested and added to the [opds capable clients](https://wiki.kavitareader.com/guides/opds) section of the wiki.

### Idea Category

Feature Enhancement

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
This is an unofficial app it looks like, it is not created by Kavita team. 

There is no point in adding it unless it's released and validated by a Kavita team member to be working. 

We will keep an eye on it and observe development.

**Comment by majora2007** [comment]
Update: I do know of the original app, WNReader which added Kavita support. It looks like they are building a dedicated client for Kavita. I will reach out to the developer and get more insight.
